modules/ocf_filehost/manifests/init.pp

```diff
@@ -5,7 +5,7 @@
 
   file {
     '/etc/exports':
-      source  => 'puppet:///modules/ocf_filehost/exports',
+      content => epp('puppet:///modules/ocf_filehost/exports'),
```

Comment:
While this works, OCF tends to use the pattern:

`content => template(ocf_filehost/exports.erb)`.

It's much more apparent what is going on.